src/memory/memory.c

```diff
@@ -710,3 +727,77 @@ bool memory_add_noise_remote_static_pubkey(const uint8_t* pubkey)
 
     return _write_chunk(CHUNK_1, chunk.bytes);
 }
+
+memory_multisig_set_by_hash_result_t memory_multisig_set_by_hash(
+    const uint8_t* hash,
+    const char* name)
+{
+    chunk_2_t chunk = {0};
+    CLEANUP_CHUNK(chunk);
+    _read_chunk(CHUNK_2, chunk_bytes);
```

Comment:
what is `chunk_bytes`?

Reply:
Same as chunk.bytes, fixed. Will fix it in the other functions in a different PR.